docs/src/main/paradox/backend-actor.md

```diff
@@ -0,0 +1,15 @@
+Backend Actor logic
+-------------------
+
+In this example, the backend only uses one basic actor. In a real system, we would have many actors interacting with each other and perhaps, multiple data stores and microservices. 
+
+An interesting side-note to add here is perhaps about when using Actors in applications like this adds value over just providing functions that would return Futures.
+In fact, if your logic is stateless and very simple request/reply style, you may not need to back it with an Actor. Actors do shine however when you need to keep some form of state and allow various requests to access something in (or *through*) an Actor. The other stellar feature of Actors that Futures would not handle is scaling-out onto a cluster very easily, byt using [Cluster Sharding](http://doc.akka.io/docs/akka/current/scala/cluster-sharding.html) or other [location-transparent](http://doc.akka.io/docs/akka/current/scala/general/remoting.html) techniques.
```

Comment:
I know, I have the same feeling about it. However, I've seen quite some PRs addressing this "issue"